src/ngraph/ops/batch_norm.cpp

```diff
@@ -148,8 +148,17 @@ void ngraph::op::BatchNorm::generate_adjoints(autodiff::Adjoints& adjoints,
     auto gamma = get_input_op(0);
     auto beta = get_input_op(1);
     auto input = get_input_op(2);
-    auto mean = std::make_shared<op::GetOutputElement>(shared_from_this(), 1);
-    auto var = std::make_shared<op::GetOutputElement>(shared_from_this(), 2);
+
```

Comment:
This needs comments about what it is doing. Is it snapping out the `GetOutputElement`?

Reply:
will do

Reply:
@diyessi added the comment

Reply:
@diyessi added another comment since I wasn't sure if you were referring to `BatchNorm::generate_adjoints` or `GetOutputElement::generate_adjoints`